processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -287,6 +287,7 @@ protected IncrementalIndex(
       if (dimSchema.getTypeName().equals(DimensionSchema.SPATIAL_TYPE_NAME)) {
         capabilities.setHasSpatialIndexes(true);
```

Comment:
Noticed above in NewSpatialDimensionSchema the const passes true for hasBitmapIndexes, 
probably need to set capabilities.setBitmapIndex(true) here.

Reply:
Moved this outside the if/else.